tasks/go.py

```diff
@@ -6,8 +6,13 @@
 import datetime
 import os
 import shutil
+import sys
+import tempfile
+import json
+from urllib.parse import urljoin
 
 import yaml
+import requests
```

Comment:
You need to add `requests` to the `requirements.txt` file. Take into account that some builders use Python 2 so if we really need this on those we will need to update those first.

Reply:
This is still an issue, see e.g. https://gitlab.ddbuild.io/DataDog/datadog-agent/-/jobs/58943404. As a first step we should add it to `requirements.txt` and check if it can be installed in all builders (probably not :()

Reply:
I opened #7850 to help move forward with this